modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/Layout.es.js

```diff
@@ -20,6 +21,7 @@ import {VariantsContext} from './VariantsContext.es';
 export function Layout({components, editable, itemPath, rows, viewMode}) {
 	const {containerElement, pageIndex} = usePage();
 	const {activePage, defaultLanguageId, pages, title} = useFormState();
+	const state = useFormState();
```

Comment:
Is this intentional to use two different instances of useFormState? If so, it seems kind of odd to me at least from a readability standpoint and debugging in the future

Reply:
I refactored this part, so we only call useFormState() one time